dev-proxy/CommandHandlers/JwtCommandHandler.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.IdentityModel.Tokens;
+using System.Globalization;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using System.Security.Principal;
+
+namespace Microsoft.DevProxy.CommandHandlers;
+
+public static class JwtCommandHandler
+{
+    public static void CreateToken(string scheme, string name, IEnumerable<string> audience, string issuer, IEnumerable<string> roles, IEnumerable<string> scopes, IEnumerable<string> claims, double validFor)
+    {
+        var jwtIssuer = new JwtIssuer(
+            issuer,
+            RandomNumberGenerator.GetBytes(32)
+        );
+
+        var options = new JwtCreatorOptions(
+            scheme,
+            name,
+            audience,
+            issuer,
+            DateTime.UtcNow,
+            DateTime.UtcNow.AddHours(validFor),
+            roles,
+            scopes,
+            ParseClaims(claims)
+        );
+
+        var jwtToken = jwtIssuer.Create(options);
+
+        var jwt = Jwt.Create(
+            scheme,
+            jwtToken,
+            new JwtSecurityTokenHandler().WriteToken(jwtToken),
+            options.Scopes,
+            options.Roles,
+            options.Claims
+        );
+
+        Console.WriteLine(jwt.Token);
+    }
+
+    private static Dictionary<string, string> ParseClaims(IEnumerable<string> claims) 
+        => claims.Select(claim => claim.Split(" ")).ToDictionary(claimParts => claimParts[0], claimParts => claimParts[1]);
```

Comment:
Since space collides with the notation for multiple argument values (`--claims value1 value2`), I suggest that we change the notation to `claim:value` which is closer to the value that's in the generated token